app/models/manageiq/providers/amazon/cloud_manager/metrics_capture.rb

```diff
@@ -64,71 +66,38 @@ def perf_collect_metrics(interval_name, start_time = nil, end_time = nil)
     start_time   = start_time.utc
 
     begin
-      perf_init_amazon
-      perf_capture_data_amazon(start_time, end_time)
+      # This is just for consistency, to produce a :connect benchmark
+      Benchmark.realtime_block(:connect) {}
```

Comment:
not sure if we can remove this completely or if we have some perf monitoring in place that check :connect benchmarks